org.omegat.OmegaT.yml

```diff
@@ -0,0 +1,69 @@
+app-id: org.omegat.OmegaT
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+
+command: run.sh
+finish-args:
+  - --filesystem=xdg-documents
+  - --socket=wayland
+  - --socket=x11
```

Comment:
1. x11 need `--share=ipc`
2. it should be `--socket=fallback-x11` if `--socket=wayland` is present.

Reply:
I don't think Java apps talk Wayland.

Reply:
That has been fixed in the current commit